django_resized/forms.py

```diff
@@ -64,7 +64,11 @@ def save(self, name, content, save=True):
 
         if DEFAULT_NORMALIZE_ROTATION:
             img = normalize_rotation(img)
-
+            
+        if self.field.force_format == 'JPEG' or 'JPG' or 'jpeg' or 'jpg':
```

Comment:
The test is wrong, it will evaluate 'JPG' if the first comparison is False.

It should be either:

`if self.field.force_format == 'JPEG' or self.field.force_format == 'JPG' or self.field.force_format == 'jpeg' or self.field.force_format == 'jpg':`

Or a better way:

`if self.field.force_format in ['JPEG', 'JPG', 'jpeg', 'jpg']:`

Reply:
Thanks Gagaro

Reply:
It is better to make left part lowercase (or uppercase) then compare:

    self.field.force_format.lower() in ('jpeg', 'jpg'):